elastic/kibana
# [Infra UI] Replace node details flyout with asset details flyout in the inventory page

Closes #161754 
Closes #166807

To make the testing and review easier I merged the old components [cleanup PR](https://github.com/jennypavlova/kibana/pull/5) into this one

## Summary
This PR replaces the old node details view with the asset details flyout

### Old
![image](https://github.com/elastic/kibana/assets/14139027/ffbead5b-6f89-4397-b1a4-2ade74f7f227)

### New 
![image](https://github.com/elastic/kibana/assets/14139027/89fa52cd-a462-499e-900a-e26c70d17791)

### Testing

1. Go to inventory
2. Click on a host in the waffle map
3. Click on any **host**
       - These changes are related only if a `Host` is selected- in the case of a pod the view shouldn't be changed: 
       ![image](https://github.com/elastic/kibana/assets/14139027/d1b90b65-1bbf-4fbf-a0c6-6b95afe6162e)

4. Check the new flyout functionality 


https://github.com/elastic/kibana/assets/14139027/3557821c-7964-466e-8514-84c2f81bc2fd

Note: the selected host should have a border like in the previous version (this I fixed in the [last commit](https://github.com/elastic/kibana/pull/166965/commits/ff4753aa06863488074fab82616951bcb4644140)) so it should be added if there is a selected node: 
<img width="1193" alt="image" src="https://github.com/elastic/kibana/assets/14139027/6646fe47-6333-435a-a5ec-248339402224">
